Game/Materials.md

```diff
@@ -0,0 +1,168 @@
+## Material
+All things are made up of materials. This usually isn't a major concern to the average adventurer, but a savvy tradesman will want to carefully select materials of varying qualities for their projects. Characters will need to source materials for anything they are [Crafting](Crafting). Materials should be simple and interchangeable withing a quality.
+
+### Material Quality
+All materials generally come in three quality levels:
+
+#### Basic
+simple materials, not very useful for crafting anything but the most basic of things.
+#### Artisan
+Well treated, high-quality materials used for proper tools and equipment.
+#### Exotic
+Impressive materials, sourced from unique locations and situations with powerful properties.
+
+### Material Types
+There are generally 8 types of materials that will be used for creating:
+
+#### Bits
+After killing a creature you can, unless it’s been annihilated, usually [harvest](Equipment#Looting) some of what are called bits. Bits are a generalization of organs, glands, bones, and other valuables. Your character would intuitively know what parts of a creature are valuable, even if you don’t yet. 
+
+##### Basic Bits
+Simple unimpressive creature viscera, not useful for much more than cooking or filler.
+
+> Blood, mundane organs, bones
+
+##### Artisan Bits
+Purposeful materials from creatures with clearly developed biological systems doing useful things.
+
+> An undead creature's icor, the claws of a powerful predator, eyes of a creature that can see in the dark
+
+##### Exotic Bits
+Similarly highly developed biological systems, but doing truly incredible things.
+
+> The regenerating heart of a golem, the blood of an elder vampire, the fire spewing lungs of a dragon.
+
+---
+
+#### Flora
+The world is full of powerful and magical materials for characters to scavenge. Some even grow in the ground. Flora can be used for all sorts of medical and chemical processes.
+
+##### Basic Flora
+Simple plants useful for herbal remedies and basic purposes.
+
+> Wild flowers, tobaco
+
+##### Artisan Flora
+Specialized plants with great properties possible to derive.
+
+> Psychadelic mushrooms, poppeys, powerful medicinal herbs
+
+##### Exotic Flora
+Plants with truley magical properties.
+
+> Glowbulb, firegrass, noxgreen
+
+---
+
+#### Metal
+Rigid workable materials, making great [Armour](Armour), [Gear](Gear), and [Weapons](Weapons). Generally harvested from deposits in the ground or reclaimed.
+
+##### Basic Metal
+Poorly sourced metal that isn't very useful for working.
+
+> Junk metal melted down, pig iron, poorly sourced metal.
+
+##### Artisan Metals
+Simple unimpressive metals that are built for purpose.
+
+> Refined iron, copper, brass.
+
+##### Exotic Metals
+Complex impressive metals with properties great for working with.
```

Comment:
Readability suggestion:  "Complex impressive metals with properties great for working with." -> "Complex, impressive metals with great working properties."

Reply:
```suggestion
Complex, impressive metals with great working properties.
```